packages/web-framework/bin/static/scripts/ti-framework.css

```
@@ -0,0 +1,156 @@
+.ti-sidebar-flyout[hidden] {
+    display: none !important;
+}
+
+.ti-sidebar-flyout {
+    /* Positioning */
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1000;
+    transform: none;
+    inset: auto;
```

Biome (2.1.2)

[error] Unexpected shorthand property inset after top



(lint/suspicious/noShorthandPropertyOverrides)